allennlp/tests/modules/seq2vec_encoders/pytorch_seq2vec_wrapper_test.py

```diff
@@ -113,3 +114,14 @@ def test_wrapper_raises_if_batch_first_is_false(self):
         with pytest.raises(ConfigurationError):
             lstm = LSTM(bidirectional=True, num_layers=3, input_size=3, hidden_size=7)
             _ = PytorchSeq2VecWrapper(lstm)
+
+    def test_wrapper_works_with_alternating_lstm(self):
+        model = PytorchSeq2VecWrapper(
+                StackedAlternatingLstm(input_size=100,
```

Comment:
No need for the test fixture to be this size, it makes them slow - just use 4,5 etc.

Reply:
Good point! Thanks.